linera-core/src/worker.rs

```diff
@@ -67,20 +66,14 @@ pub static NUM_ROUNDS_IN_BLOCK_PROPOSAL: Lazy<HistogramVec> = Lazy::new(|| {
     register_histogram_vec!(
         "num_rounds_in_block_proposal",
         "Number of rounds in block proposal",
-        // Can add labels here
         &["round_type"]
     )
     .expect("Counter can be created")
 });
 
 pub static TRANSACTION_COUNT: Lazy<IntCounterVec> = Lazy::new(|| {
-    register_int_counter_vec!(
-        "transaction_count",
-        "Transaction count",
-        // Can add labels here
-        &[]
-    )
-    .expect("Counter can be created")
+    register_int_counter_vec!("transaction_count", "Transaction count", &[])
+        .expect("Counter can be created")
```

Comment:
"should" ?

Reply:
Huh, apparently I missed some on #1135 . I'll fix it